llama/config.py

```diff
@@ -37,6 +39,7 @@
     DATA_WAREHOUSE_HOST = os.getenv("ALMA_DATA_WAREHOUSE_HOST")
     DATA_WAREHOUSE_PORT = os.getenv("ALMA_DATA_WAREHOUSE_PORT")
     DATA_WAREHOUSE_SID = os.getenv("ALMA_DATA_WAREHOUSE_SID")
+    SENTRY_DSN = None
```

Comment:
I would set this to `os.getenv("SENTRY_DSN")` so we can confirm the Sentry integration during development (see additional comment below).